spec/support/database_cleaner.rb

```diff
@@ -0,0 +1,12 @@
+# frozen_string_literal: true
+
+config.before(:suite) do
+  DatabaseCleaner.clean_with(:truncation)
+  DatabaseCleaner.strategy = :transaction
+end
+
+config.around do |example|
+  DatabaseCleaner.cleaning do
+    example.run
+  end
+end
```

Comment:
Ops, faltou o bloco:

```suggestion
RSpec.configure do |config|
  config.before(:suite) do
    DatabaseCleaner.clean_with(:truncation)
    DatabaseCleaner.strategy = :transaction
  end

  config.around do |example|
    DatabaseCleaner.cleaning do
      example.run
    end
  end
end
```